packages/esm-patient-clinical-view-app/src/case-management/workspace/case-management.resource.ts

```diff
@@ -50,11 +52,31 @@ export const useCaseManagers = () => {
 };
 
 export const useRelationshipType = () => {
-  const customRepresentation = 'custom:(uuid,display)&q=Case manager';
+  const customRepresentation = 'custom:(uuid,display,displayAIsToB,displayBIsToA)&q=Case manager';
   const url = `/ws/rest/v1/relationshiptype?v=${customRepresentation}`;
   const { data, error } = useSWRImmutable<{ data: RelationshipTypeResponse }>(url, openmrsFetch);
 
-  return { data, error };
+  const tmp: {
+    uuid: string;
+    display: string;
+    direction: string;
+  }[] = [];
```

Comment:
```suggestion
 const relations: Array<{ uuid: string; display: string; direction: string }> = [];
```
Always good to have descriptive names for variables